infrastructure/docs/docker-guide.md

```diff
@@ -0,0 +1,89 @@
+## Docker guide
```

Comment:
@gussiciliano great job here!

I would separate this into 3 files. One for docer common configuration, one for AWS and one for Heroku. What do you think?

Reply:
yes :) it is a good idea

I want to thank @truccomarcos  for the research he did on this topic :D